public/app/plugins/datasource/influxdb/components/editor/config-v2/AuthSettings.tsx

```diff
@@ -171,10 +179,38 @@ export const AuthSettings = (props: Props) => {
                   </InlineField>
                 </Box>
               </>
-            )}
+            </Box>
+          )}
+          <Box display="flex" direction="row" alignItems="center" marginBottom={2}>
+            <InlineLabel
+              style={{ width: '150px' }}
+              tooltip={'Whether credentials such as cookies or auth headers should be sent with cross-site requests.'}
+            >
+              With Credentials
+            </InlineLabel>
+            <InlineSwitch
```

Comment:
This should be tested, particularly as it fixed a bug, so that we can prevent regressions here.